mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -292,7 +294,12 @@ def _client_receive(self):
         """
         try:
             response = self._client.readline()
-            self.log.debug('Snippet received: %s', response)
+            response_log = str(response)
+            if self.verbose_logging:
+                self.log.debug('Snippet received: %s', response)
+            else:
+                self.log.debug('Snippet received: %s',
```

Comment:
the truncated log should indicate that it's truncated.
E.g. in Android logcat, truncated msgs follows "blah blah blah...[xxx bytes were truncated]"